packages/core/src/contrib/Functions.ts

```diff
@@ -332,8 +332,10 @@ export const compDict = {
   },
 
   /**
-   * Figure out which side of the rectangle `[t1, s1]` the `start->end` line is hitting, assuming that `start` is located at the rect's center and `end` is located outside the rectangle, and return the size of the relevant side. Also assuming axis-aligned rectangle. This is used for arrow placement in box-and-arrow diagrams.
-   */
+     * Figure out which side of the rectangle `[t1, s1]` the `start->end` line is hitting, assuming that `start` is located at the rect's center and `end` is located outside the rectangle, and return the size of the OTHER side. Also assuming axis-aligned rectangle. This is used for arrow placement in box-and-arrow diagrams.
```

Comment:
Maybe mark it with `@deprecated`. Not super sure if tsdoc will catch it

Reply:
Done, thanks